src/functionHandlers/notarisePdt/validateInputs/validateDocument.ts

```diff
@@ -168,4 +168,31 @@ export const validateV2Document = async (
   const validDomain = await isAuthorizedIssuer(issuerDomain, whitelistType);
   if (!validDomain)
     throw new UnrecognisedClinicError(issuerDomain, JSON.stringify(data.type));
+
+  /* 4. Validate logo (<=20KB base64 or https:// */
+  if (data.logo) {
+    const VALID_LOGO_PATTERN =
+      // Either base64 string or https URL in .png | .jpg | .jpeg format
+      /(data:image\/(png|jpg|jpeg);base64,.*)|(https:\/\/.*(.png|.jpg.jpeg))/;
```

Comment:
Please help to double-check this regex pattern. Thanks!

Reply:
Good catch! Shall I do:
```suggestion
      /(data:image\/(png|jpg|jpeg);base64,.*)|(https:\/\/.*.(png|jpg|jpeg))/;
```

Reply:
i think `(.png|.jpg.jpeg))` better, as `.` itself can be interpreted as any single character. first one also looks clearer

Reply:
Oh yes, good point. Actually I would still have to escape the `.` in all instances:
```suggestion
      /(data:image\/(png|jpg|jpeg);base64,.*)|(https:\/\/.*[.](png|jpg|jpeg))/;
```